### PR TITLE
✨ Add prettier (+hooks, lint-staged), flow, clean up code & add static type checking

### PR DESCRIPTION
**Pull request by thibmaek**
Best to go by this on a per-commit basis

This PR closes ticket #55

### CODE REVIEW

**Comment by devopsITP** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@itpocket/white-label-event-mobile-app-feature-55-dev-utils

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

**Comment by devopsITP** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@itpocket/white-label-event-mobile-app-feature-55-dev-utils

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

**Comment by devopsITP** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@itpocket/white-label-event-mobile-app-feature-55-dev-utils

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

**Comment by thibmaek** [comment]
@mikeverf would be best to rereview.

File: assets/events.json [hunk 1]
```diff
@@ -114,12 +115,12 @@
     "attendees": [
       {
         "id": "10",
-        "avatarUrl": "https://picsum.photos/300/250/?random",
+        "imageUrl": "https://picsum.photos/300/250/?random",
```
Comment (thibmaek) [inline assets/events.json]: @mikeverf best to take note of this, might have some impact on your/future code

**Comment by devopsITP** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@itpocket/white-label-event-mobile-app-feature-55-dev-utils

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

**Pull request merged**